integration-tests/runtime-tests/package.json

```diff
@@ -34,6 +34,7 @@
     "@typescript-eslint/eslint-plugin": "^5.15.0",
     "@typescript-eslint/parser": "^5.15.0",
     "assert": "^2.0.0",
+    "chai-bignumber": "^3.0.0",
```

Comment:
We're currently using BN.js for big numbers. I thought chai-bignumber is for the bignumber.js library, can you confirm this works for BN.js?

Reply:
Yes can confirm that it is working with BN.js